protected/models/Disorder.php

```diff
@@ -50,6 +51,8 @@ class Disorder extends BaseActiveRecordVersioned
     public static $SNOMED_DIABETES_TYPE_I_SET = array(46635009, 420868002);
     public static $SNOMED_DIABETES_TYPE_II_SET = array(44054006, 422014003);
 
+    public $is_ophthalmic;
```

Comment:
If you can think of an alternative, I'm all ears. But without this declaration here, the DisorderController fails to $disorder->is_ophthalmic, because it doesn't exist. Hopefully there is a way around that.